data/mods/Graphical_Overmap/overmap_terrain.json

```diff
@@ -204,6 +204,118 @@
     "sym": "â",
     "color": "brown"
   },
+  {
+    "type": "overmap_terrain",
```

Comment:
I am thinking you should be using `copy-from` and change only `sym` (and color if necessary), so other settings would be copied over from original mods.

Reply:
I see, it's so I'm not unnecessarily overwriting any changes that authors would make to their respective mods? I've cleaned up the files. Thanks for the feedback.

Reply:
Yes, GO mods only needs symbol to be changed, so any other properties would remain intact when copying.